src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_test.py

```diff
@@ -100,7 +100,7 @@ def _write_fake_units(self):
 
     def communicate(self, input_data=None):
       """Mock subprocess.Popen.communicate."""
-      if self.command[0].endswith('_fuzzer'):
+      if '/fake/build_dir/fake_fuzzer' in self.command:
```

Comment:
This functionality was already broken for 2 minijail tests. since command[0] is minijail bin.